### PR TITLE
Update webrender crate refs to reflect ownership change.

### PR DESCRIPTION
**Pull request by frewsxcv**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9733)

### CODE REVIEW

File: components/servo/Cargo.lock [hunk 1]
```diff
@@ -2228,7 +2228,22 @@ dependencies = [
 [[package]]
 name = "webrender_traits"
 version = "0.1.0"
-source = "git+https://github.com/glennw/webrender_traits#8ccc1608590d219d1df54e49f2f1e887995b4d60"
+source = "git+https://github.com/glennw/webrender_traits#11cdfa56fa7bf0f2ac8cbad8ba85e32823187a7c"
```
Comment (KiChjang) [inline components/servo/Cargo.lock]: Why is this still using glennw/webrender_traits?

Comment (frewsxcv) [inline components/servo/Cargo.lock]: Force pushed

**Comment by frewsxcv** [comment]
Ah hold on. I need to fix something here.

**Comment by frewsxcv** [comment]
Going to wait for https://github.com/servo/webrender/pull/195

**Comment by glennw** [comment]
@frewsxcv Merged that, so if you update cargo.lock in this PR it should be all good to go.

**Comment by frewsxcv** [comment]
Updated.

**Comment by frewsxcv** [comment]
r?

**Pull request merged**